ts/src/bybit.ts

```diff
@@ -6037,348 +4557,135 @@ export default class bybit extends Exchange {
-         *
-         */
+        const isUnifiedAccount = (enableUnifiedMargin || enableUnifiedAccount);
         const request = {};
-        const clientOrderId = this.safeString2 (params, 'clientOrderId', 'orderLinkId');
-        if (clientOrderId !== undefined) {
-            request['orderLinkId'] = clientOrderId;
-        } else {
-            request['orderId'] = id;
-        }
-        params = this.omit (params, [ 'clientOrderId', 'orderLinkId' ]);
-        return await this.fetchMyTrades (symbol, since, limit, this.extend (request, params));
-    }
-
-    async fetchMyUnifiedTrades (symbol: string = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
-        await this.loadMarkets ();
         let market = undefined;
-        const request = {
-            // 'symbol': market['id'],
-            // 'category': '', // Product type. spot,linear,option
-            // 'orderId': '', // Order ID
-            // 'orderLinkId': '', // User customised order ID
-            // 'baseCoin': '', // Base coin
-            // 'startTime': 0, // The start timestamp (ms)
-            // 'endTime': 0, // The end timestamp (ms)
-            // 'execType': '', // Execution type
-            // 'limit': 0, // Limit for data size per page. [1, 100]. Default: 50
-            // 'cursor': '', // Cursor. Used for pagination
-        };
+        let type = this.safeString (params, 'type', 'linear');
+        let isUsdcSettled = false;
         if (symbol !== undefined) {
             market = this.market (symbol);
+            isUsdcSettled = market['settle'] === 'USDC';
             request['symbol'] = market['id'];
-        }
-        let type = undefined;
-        [ type, params ] = this.handleMarketTypeAndParams ('fetchMyTrades', market, params);
-        if (type === 'spot') {
-            request['category'] = 'spot';
+            if (market['spot']) {
+                type = 'spot';
+            } else if (market['linear']) {
+                type = 'linear';
+            } else if (market['inverse']) {
+                type = 'inverse';
+            } else if (market['option']) {
+                type = 'option';
+            }
         } else {
-            let subType = undefined;
-            [ subType, params ] = this.handleSubTypeAndParams ('fetchMyTrades', market, params);
-            if (subType === 'inverse') {
-                throw new NotSupported (this.id + ' fetchMyTrades() does not support ' + subType + ' markets.');
+            if ((type === 'linear') || (type === 'inverse')) {
+                const baseCoin = this.safeString (params, 'baseCoin');
+                const settleCoin = this.safeString (params, 'settleCoin');
+                if ((baseCoin === undefined) && (settleCoin === undefined)) {
+                    throw new ArgumentsRequired (this.id + ' fetchOpenOrders() requires baseCoin or settleCoin argument in linear and inverse market');
```

Comment:
@sc0Vu let's try to assume reasonable defaults (USDT for instance) whenever possible to avoid throwing

Reply:
@sc0Vu besides that, let's add these parameters to the JS docs